tests/test_main.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 import unittest
+from tic_tac_toe.exceptions import *
```

Comment:
This import is unnecessary, if you look at the import below we are already importing the exceptions.